pkg/operator/connectivitycheckcontroller/manifests/0000_10-pod-network-connectivity-check.crd.yaml

```diff
@@ -0,0 +1,266 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
use the vendored version of this file from openshift/api. If necessary, you can create an update and verify script.  But we want the canonical definition in openshift/api.

Reply:
Done.